koji_wrapper/base.py

```diff
@@ -56,17 +67,14 @@ def session(self, newsession):
     def archives(self, **kwargs):
         """
         :param **kwargs: Any valid named parameter accepted by the koji client's
-            listArchives method:
-            https://pagure.io/koji/blob/master/f/hub/kojihub.py#_3950
+            listArchives method::
```

Comment:
Did you want the extra ':' here?

Reply:
I don't think so, good catch.  I was trying to fix some compile output on the docs, and thought that may have been it.  It was not, so I will remove both of these.